quarkus/server/src/main/resources/META-INF/keycloak.properties

```diff
@@ -22,16 +22,21 @@ metrics.enabled=false
 #%prod.https.certificate.file=${kc.home.dir}conf/server.crt.pem
 #%prod.https.certificate.key-file=${kc.home.dir}conf/server.key.pem
 #%prod.proxy=reencrypt
+#%prod.hostname=myhostname
 
 # Default, and insecure, and non-production grade configuration for the development profile
 %dev.http.enabled=true
+%dev.hostname.strict=false
+%dev.hostname.strict-https=false
```

Comment:
Q: How are the values set for prod mode?

Reply:
if not set, these two options are enabled by default.